### DISCUSSION TITLE
【docker部署】无法实时获取config，覆盖blog.config.js内容

### DISCUSSION THREAD

**Discussion by tzxyz** [tangly1024/NotionNext#1991] [Q&A]
**描述bug**
docker部署无法实时获取config，覆盖blog.config.js内容
**复现步骤**
1. 下载代码
2. 修改Dockerfile ARG NOTION_PAGE_ID=我的NOTION_PAGE_ID
3. docker build -t blog:latest
4. docker run 

**期望的正常结果**
1. 部署后访问网页正常显示，但似乎不是我的NOTION_PAGE_ID，因为我怎么修改都无效
2. 修改blog.config.js的NOTION_PAGE_ID后，重新docker build，生效了
3. 修改notino.so的CONFIG文档，NEXT_PUBLIC_THEME=next，不生效，修改blog.config.js重新docker build，生效

**截图**
相关的页面，应该的结果

**环境**

- 操作系统: [windows]
- 浏览器 [chrome]
- NotionNext版本 [e.g. 4.10.0]
- 主题 [例如. hexo]

**补充说明**
与问题相关的其它说明

**Comment by birdxs** [comment]
我的问题处理好了，我自己的问题，我之前修改了blog.config.js内变量名字。
最新版本4.1.3没问题
